Features/Users/users.feature

```diff
@@ -67,3 +67,10 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate that users have the "Delete" button available
+        Given I am on the Users page
```

Comment:
> Given I am on a user's page

Well, "page" is  not a term from the platform / platform ui domain. But it is at least a generic web term. Opinion, @dpobel ?

> But yes, we might shorten the sentences.

Note that the goal ain't really to shorten, but to limit the context to what matters in, well, the context.